src/app/plugins/equation.plugin.ts

```diff
@@ -19,6 +18,11 @@ export class EquationPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args: string[]): Promise<void> {
+    // Only allow in help channels
+    if (((message.channel as TextChannel).parent as CategoryChannel).name !== 'Help') {
```

Comment:
Are you sure the capitalization is correct?
Should probably do `name.toLowerCase() !== 'help'`